lib/src/ui/pages/home/news/most_popular/most_popular_news_view_entity.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:allthenews/src/ui/pages/home/news/secondary_news/news_paginated_view_entity.dart';
+import 'package:flutter/cupertino.dart';
+
+class MostPopularNewsViewEntity {
+  final NewsPaginatedViewEntities newsPaginatedViewEntities;
+  final String mostPopularNewsPageTitle;
+
+  List<NewsPaginatedViewEntity> get entities => newsPaginatedViewEntities.entities;
+
+  MostPopularNewsViewEntity({@required this.newsPaginatedViewEntities, @required this.mostPopularNewsPageTitle});
```

Comment:
czasami jest sprawdzenie assertem, a czasem nie, może jakaś konwencja?

Reply:
@required traktuję jako umożliwienie wymuszenia podania wszystkich parametrów z możliwością named params. Asercję + required z tego co widzę dodawaliśmy na ui do widgetów, jak masz jakąś propozycję co do konwencji to jestem otwarty ;)

Reply:
ja to chyba w ogóle bym nigdzie tych asercji nie dawał kiedy mamy adnotację @required, to musiałby być celowy use case, żeby do takiego parametru nulla przekazać

Reply:
ustalone :D